src/index.jsx

```diff
@@ -125,7 +125,7 @@ ReactDOM.render(
                 <ProtectedRoute
                   Component={StudentView}
                   redirectPath="/access-denied"
-                  roles={[USER_ROLE]}
+                  roles={[ADMIN_ROLE, USER_ROLE]}
```

Comment:
Make it only USER_ROLE. If you need a MT account some credentials are

email: hang.m.cao@gmail.com
pass: Hello82!